### PR TITLE
feat: add `min-issues` input

### LINKED ISSUE #3: Add an option for a min # of issues in a milestone for it to be closed
<sub>[Created in Slack](https://instantishtesting.slack.com/archives/CM8L7968G/p1588290108001900)</sub>

Right now, it's hardcoded as 3, but it would be nice to be able to specify that in the options.

### PR DESCRIPTION
**Pull request by EndBug**
Closes #3 

I've added a new input to expose the setting that controls the minimum number of issues for a milestone to be closed.
Tests are passing, even though I noticed there are no tests for the input validation, should I add some?

I've noticed that Eslint is missing some dependencies and that some others are out-of-date: we may consider updating them in a future PR

### CODE REVIEW

File: __tests__/main.test.ts [hunk 1]
```diff
@@ -55,7 +55,8 @@ function generateMilestone(
 
 const DefaultProcessorOptions: MilestoneProcessorOptions = {
   repoToken: 'none',
-  debugOnly: true
+  debugOnly: true,
+  minIssues: 3
```
Comment (marissamarym) [inline __tests__/main.test.ts:59]:
```suggestion
  minIssues: 1
```
Let's default to 1 instead

**Review by marissamarym — APPROVED** [review]
Looks great!

File: __tests__/main.test.ts (hunk 1 above)
Comment (EndBug) [reply __tests__/main.test.ts:59]: Sure, I'll update the default value in the action manifest too 👍🏻

**Review by marissamarym — APPROVED** [review]

**Comment by EndBug** [comment]
Ok, now merge conflicts are resolved, it's ready to merge

**Pull request merged**